boilerplate/app/models/AuthenticationStore.ts

```diff
@@ -0,0 +1,44 @@
+import { Instance, SnapshotOut, types } from "mobx-state-tree"
+
+export const AuthenticationStoreModel = types
+  .model("AuthenticationStore")
+  .props({
```

Comment:
```suggestion
  .model("AuthenticationStore", {
```

Reply:
I prefer this style rather than `.props({ ... })` and have been trying to use it consistently everywhere. Gafo: 4/10